Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -7,6 +7,8 @@
 #include <cmath>
 #include <cstring>
 
+#include <Core/Core.h>
```

Comment:
Dolphin-internal headers should use `""`, not `<>`. And also be part of that block two lines down.